src/foam/nanos/controller/ApplicationController.js

```diff
@@ -412,6 +423,22 @@ foam.CLASS({
         self.onUserAgentAndGroupLoaded();
         self.mementoChange();
       });
+
+      // Reload styling on theme change
+      this.onDetach(this.sub('themeChange', () => {
+        for ( const eid in this.styles ) {
+          let text = this.styles[eid];
+          for ( var i = 0 ; i < this.MACROS.length; i++ ) {
```

Comment:
Couldn't this part use the returnExpandedCSS function?

Reply:
Fixed in https://github.com/kgrgreer/foam3/pull/589/commits/1b89b36fb9c14974dc80dc8d456a1f4b87e12b94.